demisto_sdk/commands/format/tests/test_formatting_json_test.py

```diff
@@ -755,3 +755,20 @@ def test_json_run_format_exception_handling(self, format_object, mocker, capsys)
         formatter.run_format()
         stdout, _ = capsys.readouterr()
         assert 'Failed to update file my_file_path. Error: MY ERROR' in stdout
+
+    def test_set_fromversion_six_new_contributor_pack(self, pack, mocker):
```

Comment:
Let's add a test for a JSOM with an existing fromversionn key (like you had in the yml tests)